ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-standard-access-lists.j2

```diff
@@ -1,10 +1,14 @@
 {# eos - IPv6 standard access-lists #}
-{% if ipv6_standard_access_lists is defined and ipv6_standard_access_lists is not none %}
+{% if ipv6_standard_access_lists is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed

Reply:
this check is redundant and should be removed